File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/UnsubscribeEndpoint.java
package com.github.kotvertolet.youtubejextractor.models.test;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class UnsubscribeEndpoint{

	@SerializedName("channelIds")
	private List<String> channelIds;

	@SerializedName("params")
	private String params;

	public void setChannelIds(List<String> channelIds){
		this.channelIds = channelIds;
	}

	public List<String> getChannelIds(){
		return channelIds;
	}

	public void setParams(String params){
		this.params = params;
	}

	public String getParams(){
		return params;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/UnsubscribeEndpoint.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class UnsubscribeEndpoint : Parcelable, Serializable {
    @SerializedName("channelIds")
    var channelIds: List<String>? = null

    @SerializedName("params")
    var params: String? = null
    override fun toString(): String {
        return "UnsubscribeEndpoint{" +
                "channelIds = '" + channelIds + '\'' +
                ",params = '" + params + '\'' +
                "}"
    }
}